subliminal/core.py

```diff
@@ -506,7 +530,7 @@ def scan_videos(path, age=None, archives=True):
             elif archives and filename.endswith(ARCHIVE_EXTENSIONS):  # archive
                 try:
                     video = scan_archive(filepath)
-                except (NotRarFile, RarCannotExec, ValueError):  # pragma: no cover
+                except (NotRarFile, RarCannotExec, Error, ValueError):  # pragma: no cover
                     logger.exception('Error scanning archive')
```

Comment:
this is from miigotu commit and I think this is the reason as his comment:
```
# raise an exception if the rar file is broken
# must be called to avoid a potential deadlock with some broken rars
```